common/components/BalanceSidebar/AccountInfo.tsx

```diff
@@ -90,6 +104,24 @@ class AccountInfo extends React.Component<Props, State> {
           </div>
           <div className="AccountInfo-address-wrapper">
             <div className="AccountInfo-address-addr">{address}</div>
+            <CopyToClipboard onCopy={this.onCopy} text={address}>
+              <div className="copy-clipboard" title="copy to clipboard">
```

Comment:
We use SuitCSS class naming for styling, you can read about that [more here](https://github.com/MyCryptoHQ/MyCrypto/wiki/Style-Guides#classnames). Would you mind changing this to adhere to that?

Reply:
okay so the better styling would be "AccountInfo-copy-icon" ?